src/courses/templates/courses/chart_mark_distribution.html

```diff
@@ -1,105 +1,84 @@
-<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.5.0/Chart.min.js"></script>
```

Comment:
This file is pure js, so lets make it a .js static file within the courses app and include that within script tags in the html template whereever it appears.

Reply:
static doesnt work with django template tags, so not possible
![image](https://user-images.githubusercontent.com/39788517/185731327-7c4cf1d4-31c8-430b-8eec-7c8a018c6571.png)

Reply:
oh ya!